edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -24,6 +24,31 @@ var edx = edx || {};
     }
   };
 
+  /**
+   * Launch modals, handling a11y focus behavior
+   *
+   * Note: don't try to leverage this for the heartbeat; the DOM
+   * structure this depends on doesn't live everywhere that handler
+   * needs to live
+   */
+   function accessibleError(message) {
```

Comment:
The error modal this generates will be visually consistent with the modal shown on the instructions page when folks try to start the exam without having finished the desktop application setup.